tests/capi/unittest_ml_agent.cc

```diff
@@ -0,0 +1,131 @@
+/**
+ * @file        unittest_ml_agent.cc
+ * @date        16 Jul 2022
+ * @brief       Unit test for ML Agent itself
+ * @see         https://github.com/nnstreamer/api
+ * @author      Sangjung Woo <sangjung.woo@samsung.com>
+ * @bug         No known bugs
+ */
+
+#include <gio/gio.h>
+#include <gtest/gtest.h>
+#include <ml-api-internal.h>
+#include <ml-api-service.h>
```

Comment:
How about remove `<ml-api-*.h>` and test only the daemon?
Then `tests/daemon/*` might be more suitable path for this test.

Reply:
You are right! I will fix it in the next version.